app/src/main/java/com/sun/music61/data/adapter/CustomSliderAdapter.java

```diff
@@ -0,0 +1,32 @@
+package com.sun.music61.data.adapter;
+
+import com.sun.music61.R;
+import com.sun.music61.data.model.Track;
+
+import java.util.List;
+
+import ss.com.bannerslider.adapters.SliderAdapter;
+import ss.com.bannerslider.viewholder.ImageSlideViewHolder;
+
+public class CustomSliderAdapter extends SliderAdapter {
+
+    private List<Track> mBanners;
+
+    public CustomSliderAdapter(List<Track> banners) {
+        mBanners = banners;
+    }
+
+    @Override
+    public int getItemCount() {
+        return mBanners.size();
+    }
+
+    @Override
+    public void onBindImageSlide(int position, ImageSlideViewHolder imageSlideViewHolder) {
+        String urlImage = mBanners.get(position).getArtworkUrl();
+        if (urlImage != null)
+            imageSlideViewHolder.bindImageSlide(urlImage.replace("large", "t500x500"));
```

Comment:
```suggestion
            imageSlideViewHolder.bindImageSlide(mBanners.get(position).getArtworkUrl().replace("large", "t500x500"));
```
`large`
`t500x500`
hard code